lms/envs/common.py

```diff
@@ -700,6 +700,8 @@
 # Allow any XBlock in the LMS
 XBLOCK_SELECT_FUNCTION = prefer_xmodules
 
+DISABLED_XBLOCK_TYPES = ('peergrading', 'combinedopenended')
```

Comment:
Can you put this in a `ConfigurationModel`, so that we can add and remove xblocks from this list without a redeploy?